FaunaDB.Client.Test/EncoderTest.cs

```diff
@@ -397,5 +397,31 @@ public void TestStringOverride()
                 Encode(new StringOverride(testUri, testGuid))
             );
         }
+
+        [Test]
+        public void TestToStringTypes()
+        {
+            Assert.AreEqual(
+                "SetRef({\"match\": Ref(id = \"spells_by_element\", collection = Ref(id = \"indexes\")), \"terms\": water})",
+                new SetRefV(new Dictionary<string, Value>() {
+                    { "terms", StringV.Of("water") },
+                    { "match", new RefV("spells_by_element", new RefV("indexes")) }
+                }).ToString()
+            );
+
+            Assert.AreEqual("true", BooleanV.Of(true).ToString());
+            Assert.AreEqual("3.14", DoubleV.Of(3.14).ToString());
+            Assert.AreEqual("42", LongV.Of(42).ToString());
+            Assert.AreEqual("null", NullV.Instance.ToString());
+            Assert.AreEqual("Date(\"2001-01-01\")", new DateV("2001-01-01").ToString());
+            Assert.AreEqual("Time(\"2000-01-01T01:10:30.123Z\")", new TimeV("2000-01-01T01:10:30.123Z").ToString());
+            Assert.AreEqual("Arr(1, 2, 3)", ArrayV.Of(1, 2, 3).ToString());
```

Comment:
I think using brackets will be more common, `[1, 2, 3]`

Reply:
updated the `ArrayV`